src/coinbase/webhook.ts

```diff
@@ -0,0 +1,169 @@
+import { Webhook as WebhookModel, WebhookEventType, WebhookEventFilter } from "../client/api";
+import { Coinbase } from "./coinbase";
+import { InternalError } from "./errors";
+
+/**
+ * A representation of a Webhook,
+ * which provides methods to create, list, update, and delete webhooks that are used to receive notifications of specific events.
+ */
+export class Webhook {
```

Comment:
please re-export this class in `src/index.ts` so that it is importable when a developer installs our npm package

Reply:
done!